requirements/constraints.txt

```diff
@@ -8,3 +8,6 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 -c common_constraints.txt
+
+# Latest previous version has issues with doc8 because of conflicting docutils constraints
+sphinx-book-theme==0.4.0rc1
```

Comment:
FYI @feanil: when I generated the docs folder, I used just the requirements specified in the [how-to](https://docs.openedx.org/en/latest/developers/how-tos/add-sphinx-docs-to-a-repo.html?highlight=sphinx#steps). I didn't realize that libraries check rst formats [using doc8](https://github.com/openedx/openedx-events/blob/main/tox.ini#L56) which latest version has some compatibility issues with `sphinx-book-theme==0.3.3` so I had to include this constraint in my doc requirements.

Reply:
ok, thanks for the heads up.  They're pretty close to releasing sphinx-book-theme `0.4.0` so hopefully you won't need this for long.